tests/python/gaia-ui-tests/gaiatest/apps/settings/regions/display.py

```diff
@@ -9,7 +9,7 @@
 
 class Display(Base):
 
-    _page_locator = (By.ID, 'sound')
```

Comment:
That's weird that we lived with this error for nearly 2 years. This line was added on november 2013 and never modified since.

In this class, we only use it in `screen_element`, which is used in many image_compare test. Was is it working by coincidence? Like the sound panel was always open right before the display panel so we return a coherent value? pinging @npark-mozilla who has a better view on the image compare tests.